test/cli_test.py

```diff
@@ -518,14 +505,14 @@ def test_convert_write_tags(self):
 
         # We "convert" by copying the file. Setting the title simulates
         # a badly behaved converter
-        mediafile_converted = MediaFile(syspath(item.path))
+        mediafile_converted = MediaFile(item.path)
         mediafile_converted.title = "WRONG"
         mediafile_converted.save()
 
         self.runcli("alt", "update", "myexternal")
         item.load()
 
-        alt_mediafile = MediaFile(syspath(self.get_path(item)))
+        alt_mediafile = MediaFile(self.get_path(item))
```

Comment:
In contrast to the above, this passes a `pathlib.Path` to `Mediafile`.

Reply:
Like I said above, this is test code and works on both platforms so it’s fine